participants/notify.py

```diff
@@ -455,6 +484,15 @@ def handle_lifecycle_control(self, ctrl):
             okdirs = ctrl.config.get("notify", "allowed_attachment_dirs")
             self.allowed_attachment_dirs = okdirs.split()
 
+            if ctrl.config.has_option("notify", "email_whitelist"):
+                self.email_whitelist = (
+                    ctrl.config.get("notify", "email_whitelist").split())
+                self.email_whitelist = self.email_whitelist or []
```

Comment:
```suggestion
            else:
                self.email_whitelist = []
```

Reply:
I set  `self.email_whitelist = []` in `__init__` in case if config does not have option

this check is needed if config have empty option like
`email_whitelist = `
setting it to list instead of empty string

Reply:
`split()` returns empty list if the string is empty.

Reply:
oh, true
maybe just remove the whole line

Reply:
added else clause